shallow_backup/reinstall.py

```diff
@@ -73,6 +73,12 @@ def reinstall_configs_sb(configs_path):
 		elif os.path.isfile(path_to_backup):
 			copyfile(path_to_backup, dest_path)
 
+	# reinstall crontab
+	with open("{0}/crontab.txt".format(configs_path), "r") as f:
```

Comment:
Why not just run `$ crontab crontab.txt` to load in a new crontab?

Reply:
`(crontab -l ; echo \"{0}\") | sort - | uniq - | crontab -` makes sure that only non existing lines are added and existing lines aren't overwritten.